docs/DeveloperGuide.md

```diff
@@ -457,6 +459,7 @@ should be able to accomplish most of the tasks faster using commands than using
 * **Active List** / **Active Client List**: List of clients not in the archive.
 * **Active Mode**: The mode which allows users to view the active list.
 * **Archive Mode**: The mode which allows users to view the archive.
+* **Priority Indicator**: The rectangle that shows the priority of the client in the list.
```

Comment:
Maybe a screenshot here would help to show the "reactangle"?

Reply:
ok

Reply:
put 1 screenshot